packages/backend/src/accounting/psql/ledger-transfer/index.test.ts

```diff
@@ -129,51 +129,41 @@ describe('Ledger Transfer', (): void => {
 
         jest.useFakeTimers({ now })
 
-        await expect(
-          createTransfers(serviceDeps, [transfer], knex)
-        ).resolves.toEqual({
-          results: [
-            {
-              id: expect.any(String),
-              transferRef: transfer.transferRef,
-              creditAccountId: transfer.creditAccount.id,
-              debitAccountId: transfer.debitAccount.id,
-              ledger: transfer.creditAccount.ledger,
-              state: expectedState,
-              amount: transfer.amount,
-              type: transfer.type,
-              expiresAt: timeoutMs
-                ? new Date(now.getTime() + Number(timeoutMs))
-                : null,
-              createdAt: expect.any(Date),
-              updatedAt: expect.any(Date)
-            }
-          ],
-          errors: []
-        })
+        await expect(createTransfers(serviceDeps, [transfer])).resolves.toEqual(
+          {
+            results: [
+              {
+                id: expect.any(String),
+                transferRef: transfer.transferRef,
+                creditAccountId: transfer.creditAccount.id,
+                debitAccountId: transfer.debitAccount.id,
+                ledger: transfer.creditAccount.ledger,
+                state: expectedState,
+                amount: transfer.amount,
+                type: transfer.type,
+                expiresAt: timeoutMs
+                  ? new Date(now.getTime() + Number(timeoutMs))
+                  : null,
+                createdAt: expect.any(Date),
+                updatedAt: expect.any(Date)
+              }
+            ],
+            errors: []
+          }
+        )
       }
     )
 
     test('returns error if duplicate transfer (on unique transferRef constraint)', async (): Promise<void> => {
       await expect(
-        createTransfers(serviceDeps, [baseTransfer, baseTransfer], knex)
+        createTransfers(serviceDeps, [baseTransfer, baseTransfer])
       ).resolves.toEqual({
         results: [],
-        errors: [{ index: -1, error: TransferError.TransferExists }]
+        errors: [{ index: 1, error: TransferError.TransferExists }]
       })
     })
 
-    test('throws if unhandled error during insert', async (): Promise<void> => {
-      const transfer: CreateTransferArgs = {
-        ...baseTransfer
-      }
-
-      transfer.creditAccount.id = ''
-
-      await expect(
-        createTransfers(serviceDeps, [transfer], knex)
-      ).rejects.toThrow('Could not create transfer(s)')
-    })
```

Comment:
This would end up hanging the transaction for me (because of a DataError), I want to see if there would be a good solution to prevent this/add some sort of proper timeout